cromwellApiClient/src/main/scala/cromwell/api/model/WorkflowSubmission.scala

```diff
@@ -6,19 +6,22 @@ sealed trait WorkflowSubmission {
   val wdl: String
   val inputsJson: Option[String]
   val options: Option[String]
+  val customLabels: Option[String]
   val zippedImports: Option[File]
   val refreshToken: Option[String]
 }
 
 final case class WorkflowSingleSubmission(wdl: String,
                                           inputsJson: Option[String],
                                           options: Option[String],
+                                          customLabels: Option[String],
```

Comment:
ToL: from an API point of view it'd be cool if this was a `List[Label]` if somewhere we can have `case class Label(key: String, value: String)`. 
Thinking about it, we'd also need a custom JsonFormat for the label type that can serialize it into the appropriate JSON `"key": "value"`